utils.go

```diff
@@ -212,3 +214,28 @@ func getDefaultLocation(u url.URL, regionOverride string) (location string) {
 	// Default to location to 'us-east-1'.
 	return "us-east-1"
 }
+
+const letterBytes = "abcdefghijklmnopqrstuvwxyz01234569"
+const (
+	letterIdxBits = 6                    // 6 bits to represent a letter index
+	letterIdxMask = 1<<letterIdxBits - 1 // All 1-bits, as many as letterIdxBits
+	letterIdxMax  = 63 / letterIdxBits   // # of letter indices fitting in 63 bits
+)
+
+// randString generates random names and prepends them with a known prefix.
+func randString(n int, src rand.Source, prefix string) string {
```

Comment:
The randString() method was moved to utils.go because api_functional_v2_test.go uses this function as well.